modules/veteran_confirmation/spec/services/veteran_confirmation/status_service_spec.rb

```diff
@@ -1,184 +0,0 @@
-# frozen_string_literal: true
-
-require 'rails_helper'
-require 'mpi/models/mvi_profile'
-require 'mpi/service'
-
-RSpec.describe VeteranConfirmation::StatusService do
```

Comment:
I'm not familiar with how the Ruby code works, but I notice it says Veteran Confirmation here. Is removing this intentional? Seems like this is for Veteran Verification only.

Reply:
Yes, the main purpose of the PR was to remove the veteran_verification module but two veteran confirmation tests were affected by the removal of the module and failing test cases, so had to remove the status_service_spec.rb and the veteran_status_request_spec.rb

Reply:
Those depended on test items in the removed-vv module. In fact. this entire VC module should also be deleted since VC is covered by the DVP variant.